packages/react-core/src/components/Select/examples/Select.md

```diff
@@ -10,6 +10,10 @@ import CubeIcon from '@patternfly/react-icons/dist/esm/icons/cube-icon';
 
 ## Examples
 
+When passing in a value to the `menuAppendTo` prop on the Select component, passing in `document.body` should be avoided if possible as doing so can cause accessibility issues. These issues can include (but may not be limited to) being unable to enter the contents of the Select options via assistive technologies (like keyboards or screen readers).
```

Comment:
I think the content is great.
It probably does not need to be the first thing on the Select documentation.
I suggest you add it to the relevant example with the rest of the content you wrote.